app/models/miq_schedule_worker/runner.rb

```diff
@@ -170,24 +170,25 @@ def schedules_for_scheduler_role
       enqueue([:job_check_for_evm_snapshots, job_not_found_delay])
     end
 
-    # Schedule - JobProxyDispatcher#dispatch
-    # Queue a JobProxyDispatcher dispatch task at high priority unless there's already one on the queue
-    # This dispatch method goes through all pending jobs to see if there's a free proxy available to work on one of them
-    # It is very expensive to constantly do this, hence the need to ensure only one is on the queue at one time
-    scheduler.schedule_every(
-      :job_proxy_dispatcher_dispatch,
-      worker_settings[:job_proxy_dispatcher_interval]
-    ) do
-      enqueue(:job_proxy_dispatcher_dispatch)
+    scheduler.schedule_every(:container_scan_dispatcher_dispatch, worker_settings[:container_scan_dispatcher_interval]) do
```

Comment:
As silly as it sounds, can you add the comment line to each one of these (I've been trying to keep that consistent)?

```
# Schedule - ...
```